src/components/PageStepper.vue

```diff
@@ -19,7 +19,12 @@
                 <DynamicTagWrapper :tag="(isStepClickable(route.path) ? 'a' : 'span')"
                                   href="javascript:void(0);"
                                   :data-cy="getCypressValue(index)"
-                                  @click="handleClickLink(route.path)">
+                                  @click="handleClickLink(route.path)"
+                                  :role="isStepClickable(route.path) && 'link'"
+                                  :tabindex="isStepClickable(route.path) && '0'"
+                                  :disabled="isStepClickable(route.path) && 'true'"
+                                  :aria-disabled="isStepClickable(route.path) && 'true'"
```

Comment:
Oh yeah my bad, thanks you and @stephanieharrison-maximus both caught that error. I switched it up to a ternary to be a bit more clear and added a unit test for it.

For copying over, I think we can talk it over at standup I'm not 100% sure where we landed. I don't think changing these attributes should impact the other apps, but if anything is styling on them it would be a problem. I won't merge anything until everyone is on the same page